### ISSUE TITLE
 FAILURE: Build failed with an exception.

### ISSUE THREAD

**Issue by kamaravichow** [kamaravichow/iplib-android#2]
Jitpack build error

```
FAILURE: Build failed with an exception.

* Where:
Script '/script/maven-plugin.gradle' line: 2

* What went wrong:
A problem occurred evaluating script.
> Failed to apply plugin 'com.github.dcendents.android-maven'.
   > Could not create plugin of type 'AndroidMavenPlugin'.
      > Could not generate a decorated class for type AndroidMavenPlugin.
         > org/gradle/api/publication/maven/internal/MavenPomMetaInfoProvider

```

**Comment by kamaravichow** [comment]
Found a solution
https://stackoverflow.com/questions/67599438/gradle-7-and-jitpack-io-runs-into-error-during-publish

**Comment by kamaravichow** [comment]
FIXED IN v1.1

https://jitpack.io/com/github/kamaravichow/iplib-android/1.1/build.log
